src/library/math/matrix.hpp

```diff
@@ -120,32 +130,32 @@ class Matrix {
   T matrix_[R][C];  //!< Array to save the elements
 
   /**
-   * @fn is_valid_range_
+   * @fn ISValidRange
    * @brief Judge the target row/column number is in the range
    * @param [in] row: Target row number
    * @param [in] column: Target column number
    * @return True: row/column number is in the range
    */
-  inline bool is_valid_range_(size_t row, size_t column);
+  inline bool ISValidRange(size_t row, size_t column) { return (row < R && column < C); }
 };
 
 /**
- * @fn fill_up
+ * @fn FillUp
  * @brief Fill up all elements with same value
  * @param [in] m: Target matrix
  * @param [in] t: Scalar value to fill up
  */
 template <size_t R, size_t C, typename T>
-void fill_up(Matrix<R, C, T>& m, const T& t);
+void FillUp(Matrix<R, C, T>& m, const T& t);
 
 /**
- * @fn trace
- * @brief Calculate and return the trace of matrix
+ * @fn CalcTrace
+ * @brief Calculate and return the CalcTrace of matrix
```

Comment:
```suggestion
 * @brief Calculate and return the trace of matrix
```

[NITS] ここは `trace` のままでよさそうです．

Reply:
done